HostCUDA.cu

```diff
@@ -1572,11 +1572,11 @@ __global__ void nodeGravityComputation(
       poten = pot;
       idt2max = idt2;
       for (int offset = NODES_PER_BLOCK/2; offset > 0; offset /= 2) {
-        sumx += __shfl_down(sumx, offset, NODES_PER_BLOCK);
-        sumy += __shfl_down(sumy, offset, NODES_PER_BLOCK);
-        sumz += __shfl_down(sumz, offset, NODES_PER_BLOCK);      
-        poten += __shfl_down(poten, offset, NODES_PER_BLOCK);
-        idt2max = fmax(idt2max, __shfl_down(idt2max, offset, NODES_PER_BLOCK));
+        sumx += __shfl_down_sync(sumx, offset, NODES_PER_BLOCK);
```

Comment:
Looking at the documentation for __shfl_down_sync(), the first argument needs to be a thread mask, (e.g., 0xffffffff).  I'm guessing this is causing CUDA errors.